xarray/namedarray/daskmanager.py

```diff
@@ -49,9 +56,9 @@ def normalize_chunks(
             limit=limit,
             dtype=dtype,
             previous_chunks=previous_chunks,
-        )
+        )  # type: ignore
 
-    def from_array(self, data: Any, chunks, **kwargs) -> DaskArray:
+    def from_array(self, data: Any, chunks: Any, **kwargs: Any) -> Any:
```

Comment:
Is there a reason why we're setting all these types to `Any` here?

Reply:
originally, i was trying to appease `mypy` but i'm going to try to reintroduce them in a separate commit

Reply:
this has been addressed